helm/kubernetes-cluster-prep/values.yaml

```diff
@@ -0,0 +1,50 @@
+# Default values for conjur-cluster-prep.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+conjur:
+  account: "default"
+
+  # Conjur Appliance URL. (This is a required value).
+  # applianceUrl:
+
+  # Path to a Conjur certificate file. Either this value or
+  # 'certificateBase64' must be provided, but not both. Example:
+  # certificateFilePath: "files/conjur-cert.pem"
+  # certificateFilePath:
+
+  # Base64-encoded Conjur certificate file
+  # certificateBase64:
+
+authnK8s:
+  # Conjur authn-k8s authenticator ID to use for authentication.
+  # (This is a required value).
+  # authenticatorID:
```

Comment:
This can be uncommented, i.e. `  authenticatorID:`.  Sorry to keep going back and forth on this, but I know our UT will work fine with this uncommented, so better for user to make it visible.

Reply:
uncommented